assets/scripts/users/authentication.js

```diff
@@ -100,12 +100,11 @@ export async function loadSignIn () {
     )
 
     saveSignInDataLocally()
-  } else {
-    if (window.localStorage[LOCAL_STORAGE_SIGN_IN_ID]) {
-      store.dispatch(
-        setSignInData(JSON.parse(window.localStorage[LOCAL_STORAGE_SIGN_IN_ID]))
-      )
-    }
+  } else if (window.localStorage[LOCAL_STORAGE_SIGN_IN_ID]) {
+    // old login data is in localstorage but we don't have the cookies we need
+    setMode(MODES.FORCE_RELOAD_SIGN_IN)
```

Comment:
I don't think this is the appropriate error message to send them to. In addition, there doesn't seem to be a path from here to clearing and resetting cookies. Let's send them to `ERRORS.AUTH_EXPIRED`.